docker-compose.yml

```diff
@@ -10,20 +10,26 @@ services:
     network_mode: host
   test: # Like dev, but linked to redis
     image: quay.io/deis/lightweight-docker-go:v0.2.0
+    environment:
+      STORAGE_REDIS_HOST: redis
+      ASYNC_REDIS_HOST: redis
```

Comment:
Added the two vars above because when we're using compose to cobble together the environment for testing (i.e. locally) or Circle (i.e. CI), the Redis container we're linked to is addressed differently. In Circle, it's `localhost`. (Multiple containers involved in a single job there share a local network, much as do two containers in the same pod in k8s.) In compose, we have to name the link instead. These new variables let us assert come control over whether we talk to `localhost` or `redis`.